docs/roadmap.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 4
+---
+
+# Roadmap
+
+Agama is a project under constant development and priorities are re-evaluated every few weeks. This
+page offers an overview on the current state of the ever-changing development plans.
+
+## Agama 12 (latest released version)
+
+- New user interface based on PatternFly v6.
```

Comment:
np: what about linking PatternFly?

Reply:
I will do on the corresponding blog post. I'm trying to keep this relatively raw (easier for updating via copy&paste).